magento/product-community-edition/CVE-2024-34102.yaml

```
@@ -0,0 +1,22 @@
+title:     Magento 2.4.7-p1, 2.4.6-p6, 2.4.5-p8 and 2.4.4-p9 Security update.
```

I would recommend this gets renamed based on the description to explain the vulnerability:

```suggestion
title:     Arbitrary Code Execution, Improper Restriction of XML External Entity Reference (XXE) vulnerability, Server Side Request Forgery
```

---

Actualyl reviewing https://helpx.adobe.com/security/products/magento/apsb24-40.html there are a number of CVE ids listed, this one is just the code execution through xml entity reference.

```suggestion
title:     Arbitrary Code Execution through Improper Restriction of XML External Entity Reference (XXE)
```

---

Hello, 

Yes Magento security releases rarely fixes only a single CVE, I used that as reference as that is the most important one.

Should I create a file for each CVE as well? Not sure how to proceed.

---

@oliverde8 Technically the idea here is one file per CVE, so that people can review and judge issues and whether they are impacted by those specific ones individually. E.g. you may determine that this one vulnerability does not apply to your setup/config, but then maybe some of the others do. At the same time that's quite a bit of work to do for each CVE for each release, and at least they all mark the same versions as insecure, so in the end up to you if you want to add the others, but I'd definitely not claim that this issue covers more than the one CVE it references.